userdocs/profiles.dev/docs/author-docs/kustomize-and-raw.md

```diff
@@ -34,6 +34,16 @@ spec:
     # ...
 ```
 
+:::tip
+When you add local artifacts (meaning those with manifests stored in the profile repository)
+to your profile, we recommend noting that you have done
```

Comment:
Am I hallucinating or is there a weird line break here? :D

Reply:
wont be visible when rendered ;)